ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -0,0 +1,88 @@
+---
+
+- name: remove preinstalled things
+  apt:
+    name:
+      - python3-cryptography
+    state: absent
+
+- name: install deps
+  apt:
+    name:
+      - pkg-config
+      - build-essential
+      - libssl-dev
+      - python3-pip
+
+    state: present
+
+- name: install python libs
+  shell: "pip3 install PyGithub requests cryptography"
+
+- name: install rustup in userspace for root
+  shell: |
+    set -eu
+    RUSTUP_VERSION=1.24.3
+    RUSTUP_SHA="a3cb081f88a6789d104518b30d4aa410009cd08c3822a1226991d6cf0442a0f8"
+    curl --proto '=https' --tlsv1.2 -sSf -O \
+      https://raw.githubusercontent.com/rust-lang/rustup/${RUSTUP_VERSION}/rustup-init.sh
+    echo "${RUSTUP_SHA}  rustup-init.sh" | sha256sum --check --
+    sh rustup-init.sh --default-toolchain nightly -y --component rust-src
+
+- name: Configure update script service
+  copy:
+    src: team_login/
+    dest: "{{ vars_team_login_path }}"
+
+- name: Build team login cron job
+  shell: "cd {{ vars_team_login_path }} && PATH=$PATH:$HOME/.cargo/bin cargo build"
+
+- name: Enable message of the day
```

Comment:
This is not currently showing on the current instance build based on this config.